backends/arm/test/ops/test_add.py

```diff
@@ -1,32 +1,26 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
-# Copyright 2023-2024 Arm Limited and/or its affiliates.
+# Copyright 2024 Arm Limited and/or its affiliates.
 # All rights reserved.
 #
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
 import logging
-import shutil
 import unittest
 
 from typing import Optional, Tuple
 
 import torch
+from executorch.backends.arm.test import common
 from executorch.backends.arm.test.test_models import TosaProfile
 from executorch.backends.arm.test.tester.arm_tester import ArmBackendSelector, ArmTester
 from parameterized import parameterized
 
-# TODO: fixme! These globs are a temporary workaround. Reasoning:
-# Running the jobs in _unittest.yml will not work since that environment don't
-# have the vela tool, nor the tosa_reference_model tool. Hence, we need a way to
-# run what we can in that env temporarily. Long term, vela and tosa_reference_model
-# should be installed in the CI env.
-TOSA_REF_MODEL_INSTALLED = shutil.which("tosa_reference_model")
-VELA_INSTALLED = shutil.which("vela")
-
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+torch.manual_seed(42)
```

Comment:
this too?

Reply:
`torch.randn` It's used in this file.

Reply:
I guess it is ok to run test each time with a new random value? You can set the seed for debugging locally.

Reply:
I see your point. Let us address that in coming PRs.